### PR TITLE
add httpbin extension

### PR DESCRIPTION
**Pull request by thrau**
## httpbin extension

adds [httpbin](https://github.com/postmanlabs/httpbin) (https://httpbin.org) to localstack. it's served through http://httpbin.localhost.localstack.cloud:4566

Screenshot at 2023-07-27 14-33-03
Screenshot at 2023-07-27 14-34-15

For testing, follow the dev guide in the [README](https://github.com/localstack/localstack-extensions/blob/httpbin-extension/httpbin/README.md)

I had to vendor the code (distribute the source code with our extension) since it's outdated and didn't work with our version of werkzeug. i had to fix a couple of other minor things and didn't see it worth while forking the unmaintained project.
in the future, we could implement this simply as gateway routes, a lot should be compatible.

for now, the extension starts an external process and proxies the requests.

### CODE REVIEW

File: httpbin/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+Get the full httpbin experience directly in LocalStack without connecting to httpbin.org!
+
+The httpbin API is served through the hostname `http://httpbin.localhost.localstack.cloud:4566`.
+
+## Install from GitHub repository
+
+Install the extension directly from the GitHub repository by running:
+
+```bash
+localstack extensions install "git+https://github.com/localstack/localstack-httpbin-extension/#egg=localstack-httpbin-extension&subdirectory=httpbin"
```
Comment (lukqw) [inline httpbin/README.md:15]: link seems to be wrong

should be 

`"git+https://github.com/localstack/localstack-extensions/#egg=localstack-httpbin-extension&subdirectory=httpbin"`

afaict

**Review by lukqw — COMMENTED** [review]
Some comments, wasn't able to successfully launch LS with the extension.

File: httpbin/localstack_httpbin/vendor/httpbin/templates/trackingscripts.html
[the remark this answers is not in the document]
Comment (thrau) [reply httpbin/localstack_httpbin/vendor/httpbin/templates/trackingscripts.html:1]: removed

File: httpbin/localstack_httpbin/vendor/__init__.py
[the remark this answers is not in the document]
Comment (thrau) [reply httpbin/localstack_httpbin/vendor/__init__.py:1]: yes it's licensed under ISC which allows redistribution

File: httpbin/README.md (hunk 1 above)
Comment (thrau) [reply httpbin/README.md:15]: actually ideally it's just `localstack-extension-httpbin` once it's released. changed it :-)

**Pull request merged**